build/charts/antrea/values.yaml

```diff
@@ -294,6 +294,26 @@ agent:
   # resources socket is mounted into antrea-agent Pods. Setting it to an empty
   # value disables the mounting.
   kubeletRootDir: "/var/lib/kubelet"
+  antreaSysctlInit:
+    # -- Enable the sysctl override. When enabled, the init container will do the process.
+    enable: true
+    # -- Path to the sysctl configuration directory on the host.
+    hostSysctlDir: "/etc/sysctl.d"
+    # -- Mount path inside the init container for the host sysctl configuration directory.
+    # The init container writes sysctl files to this directory.
+    mountSysctlDir: "/host/etc/sysctl.d"
+    # -- Name of the sysctl configuration file created or overwritten by the init container.
+    # The filename is chosen to sort last in lexicographic order within the sysctl
+    # configuration directory, so that the settings in this file are applied after other
+    # distribution- or administrator- provided sysctl configuration files.
+    overwriteFile: "99-zzzz-antrea.conf"
```

Comment:
I am not seeing there is any need to let user to define this. Better to keep the name as a constant.

Reply:
> We are not modifying sysctl settings for interfaces that are not managed by Antrea, correct? If so, it should be fine that this configuration file is not applied last.

Will update the comment to state that a relatively high filename prefix is used to ensure the file will be applied after most default distribution- or administrator-provided sysctl configuration files.

Reply:
The comment already explains it. My point is that it seems unnecessary to make it configurable.